vendor/github.com/opencontainers/selinux/go-selinux/selinux_linux.go

```diff
@@ -370,7 +370,10 @@ func writeCon(fpath string, val string) error {
 	} else {
 		_, err = out.Write(nil)
 	}
-	return err
+	if err != nil {
+		return errors.Wrapf(err, "failed to set %s on procfs", fpath)
+	}
+	return nil
```

Comment:
This change in selinux-go seems to cause a regression on our side (introduced in https://github.com/opencontainers/selinux/pull/60)